core/src/main/java/com/linecorp/armeria/common/Cookie.java

```diff
@@ -58,21 +58,49 @@
      *
      * @param name the name of the {@link Cookie}
      * @param value the value of the {@link Cookie}
+     *
+     * @deprecated Use {@link #ofSecure(String, String)} instead to create a secure {@link Cookie}.
      */
+    @Deprecated
     static Cookie of(String name, String value) {
         return builder(name, value).build();
     }
 
+    /**
+     * Returns a newly created Cookie with secure settings. {@code "Secure"}, {@code "HttpOnly"},
```

Comment:
nit; for consistency with the other factory method `Cookie#of` 😅 

```suggestion
     * Returns a newly created {@link Cookie} with secure settings. {@code "Secure"}, {@code "HttpOnly"},
```